cmd/ocm/gcp/generate-wif-script.go

```diff
@@ -0,0 +1,65 @@
+package gcp
+
+import (
+	"log"
+
+	alphaocm "github.com/openshift-online/ocm-cli/pkg/alpha_ocm"
+	"github.com/spf13/cobra"
+)
+
+var (
+	// CreateWorkloadIdentityPoolOpts captures the options that affect creation of the workload identity pool
+	GenerateScriptOpts = options{
+		TargetDir: "",
+	}
+)
+
+func NewGenerateCommand() *cobra.Command {
+	generateScriptCmd := &cobra.Command{
+		Use:              "generate [wif-config ID]",
+		Short:            "Generate script based on a wif-config",
+		Args:             cobra.ExactArgs(1),
+		Run:              generateCreateScriptCmd,
+		PersistentPreRun: validationForGenerateCreateScriptCmd,
+	}
+
+	generateScriptCmd.PersistentFlags().StringVar(&GenerateScriptOpts.TargetDir, "output-dir", "",
+		"Directory to place generated files (defaults to current directory)")
+
+	return generateScriptCmd
+}
```

Comment:
While addressing this I realized the difference for why it's not in gcp.go. All the other commands have an extra in-between command because they are two words (like "create wif-config") and putting the extra command in gcp.go keeps things organized. The generate command doesn't have that (it's just "generate", not "generate wif-config") so there isn't anything to add to gcp.go

Reply:
sgtm